jaq-json/src/lib.rs

```diff
@@ -47,6 +47,11 @@ pub use funs::base_funs;
 pub use funs::funs;
 pub use num::Num;
 
+#[cfg(feature = "sync")]
```

Comment:
Can't we actually do `pub(crate) use alloc::sync::Arc as Rc` here and use `Rc` everywhere? That would make this PR much smaller ...

Reply:
If you are fine with it, I certainly can. I just did not want to lie about what `Rc` refers to, as many people have expectations when reading it :) If you are want it though, I will gladly change it. I have no strong opinion.